scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+##[1.47.0] - 2020-09-25
```

Comment:
I think this needs to be:

```suggestion
## [1.47.0] - 2020-09-25
```

in order for it to render as a header properly.